### DISCUSSION TITLE
Xlists 刮削異常

### DISCUSSION THREAD

**Discussion by kid0613** [metatube-community/jellyfin-plugin-metatube#137] [General]
請問不知道有沒有人發現最近人物資料刮削異常,Xlists的部分無法正常刮削。

**Comment by Cyj97** [comment]
http://localhost:8096/?redirect=XsList:30460
返回这个
{"error":{"code":403,"message":"Forbidden"}}

直接访问又能进

**Reply by kid0613** [reply]
一樣的狀況！！

**Reply by xjasonlyu** [reply]
是的，xslist似乎加了一层CF验证，导致爬虫被禁止访问了。

**Comment by Topbcy** [comment]
這有辦法解嗎? ~~~
